factory/read-config.js

```diff
@@ -6,7 +6,15 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 export default function (configPath = './config.json') {
   const fullConfigPath = path.resolve(__dirname, '../', configPath);
-  if (!fs.existsSync(fullConfigPath)) throw new Error('Config file does not exist.');
+  const fullExamplePath = path.resolve(__dirname, '../', './example-config.json');
+
+  if (!fs.existsSync(fullConfigPath)) {
+    try{
+      fs.copyFileSync(fullExamplePath, fullConfigPath);
+      throw new Error("SquadJS couldn't find any configuration file. We have generated a config file at "+fullConfigPath+". This file needs to be configured before running JquadJS again.");
```

Comment:
Use a template literal here please for consistency's stake.
```
throw new Error(`SquadJS couldn't find any configuration file. We have generated a config file at ${fullConfigPath}. This file needs to be configured before running SquadJS again.`);
```

There is also a typo in SquadJS.